pyvera/__init__.py

```diff
@@ -119,6 +119,12 @@ def get_devices(self, category_filter=''):
                 self.devices.append(VeraDimmer(item, self))
             elif (item.get('deviceInfo') and
                   item.get('deviceInfo').get('categoryName') ==
+                  'Dimmable Light'):        
+                dimmer = VeraDimmer(item, self)
+                dimmer.category = "Dimmable Switch"
```

Comment:
I can see this requires more thought with HA - will merge for now - and look to refactor myself later.